.circleci/config.yml

```diff
@@ -19,15 +19,20 @@ commands:
         type: string
     steps:
       - checkout
-      - run: |
-          sudo chown -R circleci:circleci /usr/local/bin
-          sudo chown -R circleci:circleci /usr/local/lib/<< parameters.python_version >>/site-packages
+      - run:
+          name: Grant CircleCI access to dependency locations
+          command: |
+            sudo chown -R circleci:circleci /usr/local/bin
+            sudo chown -R circleci:circleci /usr/local/lib/<< parameters.python_version >>/site-packages
       - run:
           name: Install pip setuptools
           command: python -m pip install -U "pip==21.0.1" "setuptools>=38.0" wheel
       - run:
           command: |
             sudo pip install -r test_requirements.txt
+      - run:
+          name: Pip freeze
+          command: pip freeze
```

Comment:
Why do we want this?

Reply:
It helps debug the environment at zero cost. I needed it while working on the ticket and I thought I'd leave it there. We also have it in the main repo [[1](https://github.com/quantumblacklabs/private-kedro/blob/master/.circleci/config.yml#L54), [2](https://github.com/quantumblacklabs/private-kedro/blob/master/.circleci/config.yml#L197)]